package study_1_3_1_exception;

import org.junit.runner.JUnitCore;

public class testCode {
    public static void main(String[] args) {
        JUnitCore.main("study_1_2_3_test_use_spring.UserDaoTest");
    }
}
